recipes/libattr/all/conanfile.py

```diff
@@ -0,0 +1,111 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibAttrConan(ConanFile):
```

Comment:
Needed because the recipe uses `tools.remove_files_by_mask`.
```suggestion
from conans.errors import ConanInvalidConfiguration

required_conan_version = ">=1.33.0"


class LibAttrConan(ConanFile):
```